pkgs/servers/http/unit/default.nix

```diff
@@ -2,7 +2,6 @@
 , pcre2
 , withPython2 ? false, python2
 , withPython3 ? true, python3, ncurses
-, withPHP73 ? false, php73
```

Comment:
Can add php 80 here right away?:
```diff
diff --git a/pkgs/servers/http/unit/default.nix b/pkgs/servers/http/unit/default.nix
index e6863479f54..4f4825e6fdd 100644
--- a/pkgs/servers/http/unit/default.nix
+++ b/pkgs/servers/http/unit/default.nix
@@ -2,8 +2,8 @@
 , pcre2
 , withPython2 ? false, python2
 , withPython3 ? true, python3, ncurses
-, withPHP73 ? false, php73
-, withPHP74 ? true, php74
+, withPHP74 ? false, php74
+, withPHP80 ? true, php0
 , withPerl530 ? false, perl530
 , withPerl532 ? true, perl532
 , withPerldevel ? false, perldevel
@@ -26,8 +26,8 @@ let
     fpmSupport = false;
   };

-  php73-unit = php73.override phpConfig;
   php74-unit = php74.override phpConfig;
+  php80-unit = php80.override phpConfig;

 in stdenv.mkDerivation rec {
   version = "1.22.0";
@@ -45,8 +45,8 @@ in stdenv.mkDerivation rec {
   buildInputs = [ pcre2.dev ]
     ++ optional withPython2 python2
     ++ optionals withPython3 [ python3 ncurses ]
-    ++ optional withPHP73 php73-unit
     ++ optional withPHP74 php74-unit
+    ++ optional withPHP80 php80-unit
     ++ optional withPerl530 perl530
     ++ optional withPerl532 perl532
     ++ optional withPerldevel perldevel
@@ -70,8 +70,8 @@ in stdenv.mkDerivation rec {
   postConfigure = ''
     ${optionalString withPython2    "./configure python --module=python2  --config=python2-config  --lib-path=${python2}/lib"}
     ${optionalString withPython3    "./configure python --module=python3  --config=python3-config  --lib-path=${python3}/lib"}
-    ${optionalString withPHP73      "./configure php    --module=php73    --config=${php73-unit.unwrapped.dev}/bin/php-config --lib-path=${php73-unit}/lib"}
     ${optionalString withPHP74      "./configure php    --module=php74    --config=${php74-unit.unwrapped.dev}/bin/php-config --lib-path=${php74-unit}/lib"}
+    ${optionalString withPHP80      "./configure php    --module=php80    --config=${php80-unit.unwrapped.dev}/bin/php-config --lib-path=${php80-unit}/lib"}
     ${optionalString withPerl530    "./configure perl   --module=perl530  --perl=${perl530}/bin/perl"}
     ${optionalString withPerl532    "./configure perl   --module=perl532  --perl=${perl532}/bin/perl"}
     ${optionalString withPerldevel  "./configure perl   --module=perldev  --perl=${perldevel}/bin/perl"}

```

Reply:
I've just added it and adjusted the tests as a separate commit :slightly_smiling_face: